labeling/internal/python.go

```diff
@@ -45,7 +46,7 @@ var PythonRules = []labels.Rule{
 			Key: labels.PackageManagerPipenv,
 		}
 		pipfile, _ := c.FindFile(pipenvFiles...)
-		label.Valid = pipfile != ""
+		label.Valid = pipfile != "" || fileContainsString(c, "pyproject.toml", "pipenv")
 		label.BasePath = path.Dir(pipfile)
```

Comment:
Is this base path still correct if we find a pyproject.toml?

Reply:
Ah, that's a good question. Most probably not. Pushed a fix.